### ISSUE TITLE
CheckBox: High contrast issues

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-js#53]
_From @mikewheaton on June 5, 2016 19:34_
1. Checkboxes need a hover and a focus state in high contrast mode.
2. Disabled states aren't indicated correctly. Needs to be green in HC black, and maroon in HC White.
   - ![image](https://cloud.githubusercontent.com/assets/1382445/15807690/94e16ff2-2b19-11e6-98ff-c0b6c0295f28.png)

_Copied from original issue: OfficeDev/office-ui-fabric-core#632_

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
